Library/Homebrew/rubocops/homepage.rb

```diff
@@ -90,6 +90,8 @@ def autocorrect(node)
             return if node.nil?
 
             homepage = string_content(node).dup
+            return if homepage.nil? || homepage.empty?
```

Comment:
```suggestion
            return if homepage.blank?
```

Reply:
We don't have ActiveSupport here so can't do that.

Reply:
Sorry, I was mistaken